### PR TITLE
Update docker.md

### PR DESCRIPTION
**Pull request by enexusde**

### CODE REVIEW

File: docs/devops-guide/docker.md [hunk 1]
```diff
@@ -9,27 +9,33 @@ sidebar_label: Docker
 In order to quickly run Jitsi Meet on a machine running Docker and Docker Compose,
 follow these steps:
 
-1. Download and extract the [latest release]
+1. Install git:
```
Comment (saghul) [inline docs/devops-guide/docker.md:12]: This is not needed unless you are installing the development version, which is not recommended.

**Review by saghul — CHANGES_REQUESTED** [review]

File: docs/devops-guide/docker.md [hunk 2]
```diff
@@ -9,27 +9,33 @@ sidebar_label: Docker
...
+   
+   ```bash
+   apt install git
+   ```
+
+2. Download and extract the [latest release]
     
     Alternatively, to test the latest changes clone the repository:
```
Comment (enexusde) [inline docs/devops-guide/docker.md:20]: If a alternate is mentioned the alternate should work. Either drop the alternate-suggestion or make the alternate work.

Comment (saghul) [reply docs/devops-guide/docker.md:20]: Sure, but then the command should go on the alternative, not as the first step.

**Comment by saghul** [comment]
Hey @enexusde do you intend to finish this one?

**Comment by enexusde** [comment]
> 
> 
> Hey @enexusde do you intend to finish this one?

No. Was a blocker and jitsi did not get it in-time to the candidate-collection of the tech-review. Tech-review without Jitsi is over, I have no interrest in Jitsi anymore.

**Pull request closed without merging**